hle-survey/transformation/src/test/scala/org/broadinstitute/monster/dap/dog/RoutineEnvironmentTransformationsSpec.scala

```diff
@@ -0,0 +1,786 @@
+    output1.deRoutineToysIncludeUnstuffedFabric.value shouldBe 1
+    output1.deRoutineToysIncludeRubber.value shouldBe 0
+    output1.deRoutineToysIncludeMetal.value shouldBe 1
+    output1.deRoutineToysIncludeAnimalProducts.value shouldBe 0
+    output1.deRoutineToysIncludeLatex.value shouldBe 1
+    output1.deRoutineToysIncludeRope.value shouldBe 0
+    output1.deRoutineToysIncludeTennisBalls.value shouldBe 1
+    output1.deRoutineToysIncludeSticks.value shouldBe 0
+    output1.deRoutineToysIncludeOther.value shouldBe 1
+    output1.deRoutineToysOtherDescription.value shouldBe "Soccer Ball"
+    output1.deRoutineToysHoursPerDay.value shouldBe 3
+    output1.deLicksChewsOrPlaysWithNonToys.value shouldBe true
+
+    // de_toys_yn = '1', de_toy_other_yn = '0', de_chew_other_yn = '0'
+    val example2 = Map[String, Array[String]](
+      "de_toys_yn" -> Array("1"),
+      "de_toy_plastic" -> Array("0"),
+      "de_toy_stuffed_fabric" -> Array("1"),
+      "de_toy_fabric_unstuffed" -> Array("0"),
+      "de_toy_rubber" -> Array("1"),
+      "de_toy_metal" -> Array("0"),
+      "de_toy_animal" -> Array("1"),
+      "de_toy_latex" -> Array("0"),
+      "de_toy_rope" -> Array("1"),
+      "de_toy_tennis_ball" -> Array("0"),
+      "de_toy_sticks" -> Array("1"),
+      "de_toy_other_yn" -> Array("0"),
+      "de_toys_amt" -> Array("2"),
+      "de_chew_other_yn" -> Array("0")
+    )
+    val output2 =
+      RoutineEnvironmentTransformations.mapRoutineEnvironment(RawRecord(1, example2))
+
+    output2.deRoutineToys.value shouldBe true
+    output2.deRoutineToysIncludePlastic.value shouldBe 0
+    output2.deRoutineToysIncludeStuffedFabric.value shouldBe 1
+    output2.deRoutineToysIncludeUnstuffedFabric.value shouldBe 0
+    output2.deRoutineToysIncludeRubber.value shouldBe 1
+    output2.deRoutineToysIncludeMetal.value shouldBe 0
+    output2.deRoutineToysIncludeAnimalProducts.value shouldBe 1
+    output2.deRoutineToysIncludeLatex.value shouldBe 0
+    output2.deRoutineToysIncludeRope.value shouldBe 1
+    output2.deRoutineToysIncludeTennisBalls.value shouldBe 0
+    output2.deRoutineToysIncludeSticks.value shouldBe 1
+    output2.deRoutineToysIncludeOther.value shouldBe 0
+    output2.deRoutineToysOtherDescription shouldBe None
+    output2.deRoutineToysHoursPerDay.value shouldBe 2
+    output2.deLicksChewsOrPlaysWithNonToys.value shouldBe false
+
+    // de_toys_yn = '0', de_chew_other_yn = '1'
+    val example3 = Map[String, Array[String]](
+      "de_toys_yn" -> Array("0"),
+      "de_chew_other_yn" -> Array("1")
+    )
+    val output3 =
+      RoutineEnvironmentTransformations.mapRoutineEnvironment(RawRecord(1, example3))
+
+    output3.deRoutineToys.value shouldBe false
+    output3.deLicksChewsOrPlaysWithNonToys.value shouldBe true
+
+    // de_toys_yn = '0', de_chew_other_yn = '0'
+    val example4 = Map[String, Array[String]](
+      "de_toys_yn" -> Array("0"),
+      "de_chew_other_yn" -> Array("0")
+    )
```

Comment:
I feel like this test case is not that useful, should I remove it?

Reply:
If you think we have good enough coverage without it, sure